andela/health-checks-sparta
# Unresolved Stuff Dashboard

### What does this branch do?
- Adds a feature to allow users view their unresolved checks.

### How do I test this branch?
- Log into health checks and select the UNRESOLVED tab to view your unresolved checks.
- You can also run the tests for this feature on the terminal using

     `python manage.py test hc.front.tests.test_my_checks.MyChecksTestCase.test_it_shows_unresolved` 
     
     and you should get the result:

    `Creating test database for alias 'default'...`

    `.`

    `----------------------------------------------------------------------`

    `Ran 1 test in 0.091s`

    `OK`

    `Destroying test database for alias 'default'...`
